src/Microsoft.AspNetCore.Blazor.Templates/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/BlazorHosted-CSharp.Server.csproj

```diff
@@ -8,12 +8,8 @@
     <LangVersion>7.3</LangVersion>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(Framework)' != 'netcoreapp2.0'">
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Blazor.Server" Version="$(TemplateBlazorPackageVersion)" />
-  </ItemGroup>
-  <ItemGroup Condition="'$(Framework)' == 'netcoreapp2.0'">
-    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0" />
+  <ItemGroup Condition="'$(Framework)' == 'netcoreapp2.1'">
```

Comment:
Could we drop the `Condition` here now there's only going to be one choice for this?

Reply:
I left it here because it will just come back when we do the next big push. If you think it's important I can take it out.

Reply:
OK, that makes sense. Reasonable to leave it then.